tsMuxer/iso_writer.cpp

```diff
@@ -333,8 +333,8 @@ bool FileEntryInfo::isFile() const { return m_fileType == FileType_File || m_fil
 FileEntryInfo::~FileEntryInfo()
 {
     delete[] m_sectorBuffer;
-    for (int i = 0; i < m_files.size(); ++i) delete m_files[i];
-    for (int i = 0; i < m_subDirs.size(); ++i) delete m_subDirs[i];
+    for (size_t i = 0; i < m_files.size(); ++i) delete m_files[i];
+    for (size_t i = 0; i < m_subDirs.size(); ++i) delete m_subDirs[i];
```

Comment:
ranged-for

Reply:
Wouldn't the following do the same thing:
```
m_files.clear();
m_subDirs.clear();
```